Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -125,6 +125,7 @@
         "nMeasurementsMin",
         "maxHoles",
         "maxOutliers",
+        "maxHolesAndOutliers",
```

Comment:
need to update the defaults below:
`defaults=[(None, None)] * 7 + [None] * 7,`

Reply:
otherwise I get a Python error with full_chain_itk.py (TrackSelectorConfig missing 1 required positional argument: 'loc0').